test_common/config.py

```diff
@@ -20,3 +20,8 @@
 MINIO_MC_EXE_PATH = Path(TEST_CFG.get("test.minio.mc.exe"))
 TEMP_DIR       = Path(TEST_CFG.get("test.temp.dir"))
 TEMP_DIR_KEEP  = TEST_CFG.get("test.temp.dir.keep") == "true"
+
+# Don't use these in automated tests, would require putting creds into github which would be
+# a huge pain since they expire frequently
+SFAPI_CREDS_FILE_PATH = Path(TEST_CFG.get("test.sfapicreds.path"))
+SFAPI_CREDS_USER = TEST_CFG.get("test.sfapicreds.user")
```

Comment:
Unrelated to this line but 2 unit tests consistently failed locally for me. May or may not worth to investigate. Since GHA tests pass, I am ok to move forward. 

```
======================== 2 failed, 47 passed in 21.09s =========================
PASSEDPASSEDPASSEDPASSEDAdded `minio_controller` successfully.
PASSEDPASSEDPASSEDPASSEDPASSEDFAILED
s3/s3_remote_test.py:224 (test_upload_presigned_url)
None != 'T/xSCA=='

Expected :'T/xSCA=='
Actual   :None
<Click to see difference>

minio = <controllers.minio.MinioController object at 0x10b188210>

    @pytest.mark.asyncio
    async def test_upload_presigned_url(minio):
        await minio.clean()  # couldn't get this to work as a fixture
        await minio.create_bucket("test-bucket")
    
        s3c = await _client(minio)
        url = (await s3c.presign_post_urls(S3Paths(["test-bucket/foo/myfile"])))[0]
        urlcrc = (await s3c.presign_post_urls(S3Paths(["test-bucket/bar/myfilecrc"])))[0]
        urlcrc.fields["x-amz-checksum-crc32"] = "T/xSCA=="
        async with aiohttp.ClientSession() as sess:
            await upload_presigned_url(sess, url.url, url.fields, TEST_RAND10KB, timeout_sec=5)
            await upload_presigned_url(sess, urlcrc.url, urlcrc.fields, TEST_RAND10KB)
        with open(TEST_RAND10KB, "rb") as f:
            expectedfile = f.read()
        objdata = await minio.get_object("test-bucket", "foo/myfile")
        objdatacrc = await minio.get_object("test-bucket", "bar/myfilecrc")
    
        for o, crc in ((objdata, None), (objdatacrc, "T/xSCA==")):
            body = await o["Body"].read()
            assert body == expectedfile
            assert o["ETag"] == '"3291fbb392f6fad06dbf331dfb74da81"'
>           assert o.get("ChecksumCRC32") == crc
E           AssertionError: assert None == 'T/xSCA=='
E            +  where None = <built-in method get of dict object at 0x10c472e00>('ChecksumCRC32')
E            +    where <built-in method get of dict object at 0x10c472e00> = {'AcceptRanges': 'bytes', 'Body': <StreamingBody at 0x10c473180 for ClientResponse at 0x10c470a90>, 'ContentLength': 10240, 'ContentType': 'application/octet-stream', ...}.get

s3/s3_remote_test.py:246: AssertionError
FAILED
s3/s3_remote_test.py:248 (test_upload_presigned_url_with_crc_and_insecure_ssl)
minio = <controllers.minio.MinioController object at 0x10b188210>

    @pytest.mark.asyncio
    async def test_upload_presigned_url_with_crc_and_insecure_ssl(minio):
        await minio.clean()  # couldn't get this to work as a fixture
        await minio.create_bucket("test-bucket")
    
        s3c = await _client(minio)
        url = (await s3c.presign_post_urls(S3Paths(["test-bucket/foo/myfile"])))[0]
        async with aiohttp.ClientSession() as sess:
            # There's no a lot to test with insecure ssl other than it doesn't break things
            # Unless we want to get really crazy and set up Minio with a SSC in the tests. We don't
            await upload_presigned_url_with_crc32(
                sess, url.url, url.fields, TEST_RAND1KB, insecure_ssl=True, timeout_sec=5)
        with open(TEST_RAND1KB, "rb") as f:
            expectedfile = f.read()
        objdata = await minio.get_object("test-bucket", "foo/myfile")
    
        body = await objdata["Body"].read()
        assert body == expectedfile
        assert objdata["ETag"] == '"b10278db14633f102103c5e9d75c0af0"'
>       assert objdata["ChecksumCRC32"] == "7Zpusw=="
E       KeyError: 'ChecksumCRC32'

s3/s3_remote_test.py:268: KeyError
PASSEDPASSEDPASSED
Process finished with exit code 1
```

Reply:
I'm guessing you need to update your Minio version